torch/autograd/gradcheck.py

```diff
@@ -1289,6 +1290,7 @@ def _gradcheck_helper(func, inputs, eps, atol, rtol, check_sparse_nnz, nondet_to
 def gradgradcheck(
     func: Callable[..., _TensorOrTensors],  # See Note [VarArg of Tensors]
     inputs: _TensorOrTensors,
+    *,
```

Comment:
I think you want to keep grad_outputs positional no?
We want to make sure we break as few users as possible.

Reply:
Happy to swap to make positional and everything else after kwarg only?

Reply:
yes